e2e-tests/monitoring/01-assert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 300
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: monitoring
+status:
+  observedGeneration: 1
+  replicas: 1
+  readyReplicas: 1
+  currentReplicas: 1
+  updatedReplicas: 1
+  collisionCount: 0
```

Comment:
service is also worth checking

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/757e92c61cb0691b3713cd46d05a7091094ec2e7